Cargo.toml

```diff
@@ -16,6 +16,10 @@ keywords = [
 	"webassembly",
 ]
 
+[[bin]]
+name = "gluesql"
+path = "cli"
```

Comment:
`cli` is a directory.
First of all, let's upload only the workspace version in this PR.
Even if you specify `cli/src/main.rs`, you will have to import clap (+ other crates) because the dependency is `gluesql`.

How about making the cli into a library later and going in the direction of holding main in gluesql?

Reply:
> How about making the cli into a library later and going in the direction of holding main in gluesql?

This makes sense, but we still need to update `memory-storage` and `sled-storage` Cargo.toml versions.

> cli is a directory.
First of all, let's upload only the workspace version in this PR.
Even if you specify cli/src/main.rs, you will have to import clap (+ other crates) because the dependency is gluesql.

I think I don't get the point, can you explain more details about this?

Reply:
1. I agree with the version update of storages workspaces.
2. bin/path is where you enter the path. The reason why it is not currently being built is that the directory is entered in the bin/path, not the main file (in terms of execution context).

I think it would be nice to separate the 2 from another PR.

Reply:
About "1.", there was an another thing that I misunderstood.
Currently, default implementation of `version()` in `Metadata` is in `core` workspace, so `core/Cargo.toml` version was supposed to be the same as the root, not storages.